src/Spain/Mews.Fiscalizations.Spain/Model/Response/ResponseResult.cs

```diff
@@ -2,6 +2,15 @@
 
 namespace Mews.Fiscalizations.Spain.Model.Response
 {
+    public static class ResponseResult
+    {
+        public static ResponseResult<T> Success<T>(T result) where T : class => new ResponseResult<T>(successResult: result);
+
+        public static ResponseResult<T> Error<T>(ErrorResult error) where T : class => new ResponseResult<T>(errorResult: error);
+
+        public static ResponseResult<T> Error<T>(string code, string message) where T : class => new ResponseResult<T>(errorResult: new ErrorResult(code, message));
```

Comment:
I kinda like this. But the lines are too long and the where doesn't help this. What do you think @marektresnak?

Reply:
I do like it as well. Maybe we could just split to two lines? The long lines are difficult to read on some places (like here, in Github).